packages/block-library/src/image/edit.js

```diff
@@ -87,6 +87,20 @@ function hasDefaultSize( image, defaultSize ) {
 	);
 }
 
+/**
+ * Checks if a media attachment object has been "destroyed",
+ * that is, removed from the media library. The core Media Library
+ * adds a `destroyed` property to a deleted attachment object in the media collection.
+ *
+ * @param {number} id The attachment id.
+ *
+ * @return {boolean} Whether the image has been destroyed.
+ */
+function isMediaDestroyed( id ) {
+	const attachment = wp?.media?.attachment( id ) || {};
```

Comment:
Not sure what the 'standard' approach is as this file already used `/* global wp */` but the other [use of this in media related file ](https://github.com/WordPress/gutenberg/blob/trunk/packages/media-utils/src/components/media-upload/index.js#L12)uses `const { wp } = window;`

Reply:
I followed the existing examples in `image/edit.js` but now I look, `const { wp } = window;` seems to be used more often. I'm also not sure about this one either.